include/thread_pool/rouser.hpp

```diff
@@ -124,8 +124,13 @@ inline void Rouser::start(std::vector<std::unique_ptr<Worker<Task, Queue>>>& wor
 
 inline void Rouser::stop()
 {
+    if (!m_started_flag.load(std::memory_order_acquire))
```

Comment:
Actually, I think calling stop() shouldn't throw... in general there is an intractable race in this scenario, so would prefer to always allow stop() to succeed if already stopped (since there is no cost or allocation of resources)

Reply:
Reverted!